src/chrome/locale/ar/https-everywhere.dtd

```diff
@@ -8,12 +8,14 @@
 <!ENTITY https-everywhere.about.contribute  "إن أعجبتك إضافة HTTPS Everywhere، فكر بـ">
 <!ENTITY https-everywhere.about.donate_tor "التبرع لتور">
 <!ENTITY https-everywhere.about.tor_lang_code "en">
+<!ENTITY https-everywhere.about.or "or">
```

Comment:
I believe you can leave these entries absent until they are translated in Transifex. HTTPS Everywhere will default to the English version if there is no available translation.